src/generic/armcm_boot.c

```diff
@@ -47,6 +75,7 @@ ResetHandler(void)
     //__libc_init_array();
 
     // Run the main board specific code
+    __enable_irq();
```

Comment:
FWIW, might be more clear to move the irq enable up to just after the irq setup code completes.